generators/app/templates/webapp/model/formatter.ts

```diff
@@ -1,19 +0,0 @@
-export default {

```

Comment:
I know I mentioned the unused formatter today, but I wonder whether we should keep (and use!) it because people will need it soon.